conans/test/unittests/tools/scm/test_git_get_commit.py

```diff
@@ -35,11 +35,17 @@ def test_change_branch_in_root_commit():
 def test_multi_folder_repo():
     c = TestClient()
     conanfile = MockConanfile({})
-    c.save({"lib_a/conanfile.py": ""})
+    c.save({"conanfile.py": ""})
```

Comment:
If this test was not flawed by design, it is much better to leave it as-is, and then create a new test, even if it is very similar to the existing one, with small variations.

Reply:
Sure. I started that way and noticed I was duplicating a lot of setup.

Reply:
Yes, I know, but in those cases we prefer to do it in 2 steps:
- First add new tests, so it is evident that existing tests are not altered. Get approval and merge the PR
- Do a second PR which is a pure refactor of tests, extracting the common parts.

Just a detail, but as "not breaking" is important for us, we like to do it this way if possible. Thanks!